From fed886e88e36e3b82582c00d6c1e46b237904f39 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 26 Dec 2017 17:58:20 +0100
Subject: [PATCH 1/1] efi_loader: address of the simple file system protocol

When installing the the simple file system protocol we have to path
the address of the structure and not the address of a pointer ot the
structure.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_disk.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/efi_loader/efi_disk.c b/lib/efi_loader/efi_disk.c
index bb47b88e3a..0ce7a1befc 100644
--- a/lib/efi_loader/efi_disk.c
+++ b/lib/efi_loader/efi_disk.c
@@ -255,7 +255,7 @@ static struct efi_disk_obj *efi_disk_add_dev(
 							 diskobj->dp);
 		ret = efi_add_protocol(diskobj->parent.handle,
 				       &efi_simple_file_system_protocol_guid,
-				       &diskobj->volume);
+				       diskobj->volume);
 		if (ret != EFI_SUCCESS)
 			goto out_of_memory;
 	}
-- 
2.14.2

